test/AreaEstimator/utest_area_estimator.cpp

```diff
@@ -0,0 +1,151 @@
+#include <gtest/gtest.h>
+
+#include "area_estimator_test_utils.h"
+
+Rectangle cell = read_cell(1, -1, -1, 1);
+
+TEST(AreaEstimator_SmokeTesting, LR_SideAligned_ThroughCenter_Empty) {
```

Comment:
Well, I've made a mistake in the naming pattern application example provided is earlier comment.
The extra (optional) info should be the last name element, occupancy should be mentioned in the first element. So test's name should be Empty_LR_SideAligned_PassesCenter. Apply the same changes for the names below.